.github/workflows/test.yml

```diff
@@ -22,4 +22,4 @@ jobs:
           DB_NAME: ${{ secrets.DB_NAME }}
           DB_USER: ${{ secrets.DB_USER }}
           DB_PASSWORD: ${{ secrets.DB_PASSWORD }}
-        run: mvn test -DDB_HOST=${DB_HOST} -DDB_NAME=${DB_NAME} -DDB_USER=${DB_USER} -DDB_PASSWORD=${DB_PASSWORD}
+        run: mvn test -no-transfer-progress
```

Comment:
what is no-transfer-progress needed for?

Reply:
Just cleans up the download logs and only shows you the test run